internal/core/attribute.cpp

```diff
@@ -49,9 +49,11 @@ float MaxAt::compute(
     return segment.sample_position_at(max_index);
 }
 
-float MaxAbs::compute(
+namespace {
+
+double maxAbs(
```

Comment:
Note that in this c++ file for function names we follow snake naming convention, so a more valid name for a function would have been max_abs or maxabs

Reply:
done 
https://github.com/equinor/vds-slice/pull/214/commits/15e91b991d3ddd90e96058fbbb5769827b1c2f08#diff-0a24dd0852852a74dc874fdfcbb7826193dfa983b4b02f34f87d06fb28e91a40R54